redis_feature_store.js

```diff
@@ -0,0 +1,169 @@
+var redis = require('redis'),
+    NodeCache = require( "node-cache" );
+
+
+function RedisFeatureStore(redis_opts, cache_ttl) {
```

Comment:
@apucacao this is an optional submodule, but I'm not sure what the preferred pattern is for importing things like this. Right now, my driver does it with the following:

```
var RedisFeatureStore = require('ldclient-node/redis_feature_store');
```

but it sounds like that's not idiomatic? What's the preferred way to do this?